typegate/engine/Cargo.toml

```diff
@@ -45,3 +49,5 @@ env_logger.workspace = true
 
 [build-dependencies]
 shadow-rs.workspace = true
+# we only need deno_runtime but this way, we avoid
```

Comment:
I am not clear on that one, can you explain?

Reply:
Oof, stale comment. I was experimenting on which crate to put the build script at first.